### PR TITLE
Fix passing the config file as an argument to the docker image.

### LINKED ISSUE #1: GHA plugin does not respect config parameter (with.config)
**The problem**

When providing an alternative configuration file, using the .with.config as outlined in the documentation;
```
      # Run trojansourcedetector
      - name: Trojan Source Detector
        uses: haveyoudebuggedit/trojansourcedetector@v1
        with:
          config: path/to/config/file
```

the observed behavior is that this configuration is not used at all and the plugin will always try to load .trojansourcedetector.json from your repository.

**Expected behavior**
Have the GHA plugin using the provided path/to/config/file.

**The solution**
I believe in the action.yml one step is omitted, which would pass the config input as an argument to the docker image. I will attach a PR.

### PR DESCRIPTION
**Pull request by jortkoopmans**
## Please describe the change you are making

Fixes #1

### CODE REVIEW

**Comment by janosdebugs** [comment]
Hey @jortkoopmans awesome, thank you!

Please note, this repository is licensed under the [Unlicense](https://github.com/haveyoudebuggedit/trojansourcedetector/blob/main/LICENSE.md). Please read the license and confirm that you are ok with your contribution under this license.

**Comment by jortkoopmans** [comment]
Hi @janosdebugs , yes I'm totally fine with the Unlicense license :+1: .

File: action.yml [hunk 1]
```diff
@@ -11,3 +11,5 @@ inputs:
 runs:
   using: 'docker'
   image: 'Dockerfile'
+  args:
+    - ${{ inputs.config }}
```
Comment (janosdebugs) [inline action.yml:15]: This won't work, the config needs to be passed as `-config path/to/file`. I'll have to research how to do that in GitHub Actions.

Comment (janosdebugs) [reply action.yml:15]: Nevermind, I should read my own code. This is good. :D

**Review by janosdebugs — APPROVED** [review]

**Pull request merged**